### PR TITLE
Norway roundtrip (nearly) finished!

### PR DESCRIPTION
**Pull request by badboy**
Can it handle UTF-8?

### CODE REVIEW

**Comment by skade** [comment]
I'm not sure why giphy gave me this for "unicode", but hey.

http://45.media.tumblr.com/b9278706921eccee26080f60caa83d43/tumblr_mxg9v5yHEp1sicwp8o2_1280.gif

**Comment by bascht** [comment]
I don't know about you, but this merge already feels festive, doesn't it?

File: config/travellers/badboy.yml [hunk 1]
```diff
@@ -31,3 +31,8 @@ hasbeen:
     - "Stavanger"
     - "London"
     - "Budapest"
+    - "Bodø"
```
Comment (bascht) [inline config/travellers/badboy.yml]: # Mr @bitboxer for mayor! Bodø 2016!

Comment (schultyy) [inline config/travellers/badboy.yml]: Vote for Quimby!

Comment (bitboxer) [inline config/travellers/badboy.yml]: I will just leave this here: https://www.wikiwand.com/en/Bodo_people :wink:

**Pull request merged**